tdp/core/models/test_db.py

```diff
@@ -2,47 +2,98 @@
 # SPDX-License-Identifier: Apache-2.0
 
 import logging
-from datetime import datetime
+from datetime import datetime, timedelta
 
 import pytest
-from sqlalchemy import create_engine
+from sqlalchemy import Table, create_engine
 from sqlalchemy.orm import sessionmaker
+from sqlalchemy.orm.session import Session
 
-from tdp.core.models.base import Base
-from tdp.core.models.deployment_log import DeploymentLog
-from tdp.core.models.operation_log import OperationLog
+from .base import Base
+from .deployment_log import DeploymentLog
+from .operation_log import OperationLog
+from .service_component_log import ServiceComponentLog
 
 logger = logging.getLogger("tdp").getChild("test_db")
 
 
-@pytest.fixture(scope="session")
-def session_class():
+def empty_tables(session: Session):
```

Comment:
`clear_tables` is better. `empty` is generaly used to check if something is empty but not for removing items.